.github/workflows/build_binary.yml

```diff
@@ -23,6 +23,11 @@ jobs:
           apt-get update
           apt-get install -y --no-install-recommends git ca-certificates gcc libc6-dev curl make zip
 
+      - name: Install protoc
+        uses: arduino/setup-protoc@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
Don't think these are actually needed, the published binaries are built without `processing` and if it is necessary, it shouldn't be. Same way these binaries don't package a redis and kafka client.